### PR TITLE
Order sdk part of main.yml

### PR DESCRIPTION
**Pull request by wonhyeongseo**
resolve https://github.com/OpenBB-finance/OpenBBTerminal/issues/2780

# Description

- [x] Summary of the change / bug fix.
Alphabetically order main.yml by modifying `docs/generate.py`. Opted for the simpler strategy of just sorting the `os.walk` on-the-fly.
image
Then it was a matter of generating the docs and hugo website.
```bash
# ~/OpenBBTerminal/docs
rm -rf code && sphinx-apidoc ../openbb_terminal/ -o ./code
make html
cd .. # ~/OpenBBTerminal
python docs/generate.py
cd website # ~/OpenBBTerminal/website
hugo server -D
```

- [x] Link # issue, if applicable.
https://github.com/OpenBB-finance/OpenBBTerminal/issues/2780
- [x] Screenshot of the feature or the bug before/after fix, if applicable.

before | after
--------|--------
image | image

- [x] Relevant motivation and context.
I wanted to help out and finish the job.
- [x] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes. 
* Provide instructions so we can reproduce. 
* Please also list any relevant details for your test configuration.
- [x] Make sure affected commands still run in terminal
- [x] Ensure the api still works
- [x] Check any related reports

# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by wonhyeongseo** [comment]
Hello @piiq ,@DidierRLopes how was your weekend? Just wondering are there any updates that should be made? Thanks.

**Comment by piiq** [comment]
Thanks for the contribution @wonhyeongseo and thanks @colin99d for volunteering to be a reviewer here
Let me know if you need any assistance

**Comment by piiq** [comment]
@wonhyeongseo is there anything specific you want to achieve by becoming a github org member?

At the moment we don't have a framework to add outside contributors to the github org. There was a discussion about giving distinguished contributors some specific status that stands out and gives more rights on the repo, but this has not been moving forward. The reason for that is that the contributors that stick with the project (contribute valuable changes over a long period of time) end up joining the team. So at the moment all github org members have some sort of contractual relationships with the company.

**Review by wonhyeongseo — COMMENTED** [review]
Hello, @colin99d thank you again for your review. 😄 I like the warning message on `docs/generate.py`. 
1. One small thing that seems odd, `stocks/_index.md` has been erased. Is this intended?
2. Also as this PR has merge conflicts again, would it be better for you to review if I start a new PR with your updates?

Hello, @piiq thank you for your response. There wasn't a goal, just wanted to help more at the time. I went through your blogs and careers page after writing this PR, and thought that working here would be great. I will keep making PRs regardless, but may you please guide me to what makes a contribution valuable (note-worthy)?

**Comment by DidierRLopes** [comment]
> wonhyeongseo

hey @wonhyeongseo! Thanks for asking, had a great weekend. Hope you too.

Sorry for the delay on our side. @colin99d will have a look at this as he worked on the sdk documentation 🙏🏽

File: website/content/sdk/stocks/_index.md [hunk 1]
```diff
@@ -1,67 +1,6 @@
 ---
-title: Introduction to the Stocks Menu
-keywords: "stocks, stock, options, option, call, put, earnings, calendar, how-to, guide, scripts, fundamental, analysis, technical, behavioural, analyst, equity, research, api, sdk, application, python, notebook, jupyter"
-excerpt: "This guide introduces the Stocks menu in the context of the OpenBB SDK."
+title: stocks
```
[the remark this answers is not in the document]
Comment (colin99d) [reply website/content/sdk/stocks/_index.md:2]: Sorry my git is a complete mess I am still trying to fix it. The merge conflicts are brutal.

**Comment by colin99d** [comment]
> Hello, @colin99d thank you again for your review. 😄 I like the warning message on `docs/generate.py`.
> 
> 1. One small thing that seems odd, `stocks/_index.md` has been erased. Is this intended?
> 2. Also as this PR has merge conflicts again, would it be better for you to review if I start a new PR with your updates?
> 
> Hello, @piiq thank you for your response. There wasn't a goal, just wanted to help more at the time. I went through your blogs and careers page after writing this PR, and thought that working here would be great. I will keep making PRs regardless, but may you please guide me to what makes a contribution valuable (note-worthy)?

I am almost done with merge conflicts

**Review by colin99d — APPROVED** [review]

**Comment by colin99d** [comment]
@wonhyeongseo would love to work together more closely! Feel free to add me on Discord Charlestons#0746.

**Pull request merged**